docs/guides/using_ocio/compatible_software.rst

```diff
@@ -336,18 +336,14 @@ PhotoFlow supports OCIO via a dedicated tool that can load a given configuration
 Website : `<https://github.com/aferrero2707/PhotoFlow>`__
 
 
-Photoshop (beta)
+Photoshop
 ****************
```

Comment:
Very minor, but a warning is generated if the length of the ***** line doesn't match the length of the text above it.